pandas/tests/series/test_ufunc.py

```diff
@@ -0,0 +1,152 @@
+import string
+
+import numpy as np
+import pytest
+
+import pandas as pd
+import pandas.util.testing as tm
+
+UNARY_UFUNCS = [np.positive, np.floor, np.exp]
+BINARY_UFUNCS = [np.add, np.logaddexp]  # -> dunder op
```

Comment:
whoops, this comment is supposed to be for `np.add`.

Reply:
Any reason not to go all out and use the full list of ufuncs here?

Reply:
Three reasons

1. I couldn't find a fixture with all the ufuncs (I thought we had one, but apparently not)
2. I have to treat binary input and binary output separately, so I need separate tests anyway
3. I'm mostly concerned with the dispatching logic. I don't think the marginal value of testing `np.negative` is high, if we're already testing `np.positive`